subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1651,14 +1523,10 @@ static void light_ctrl_srv_reset(struct bt_mesh_model *model)
 {
 	struct bt_mesh_light_ctrl_srv *srv = model->user_data;
 	struct bt_mesh_light_ctrl_srv_cfg cfg = BT_MESH_LIGHT_CTRL_SRV_CFG_INIT;
-#if CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG
 	struct bt_mesh_light_ctrl_srv_reg_cfg reg_cfg = BT_MESH_LIGHT_CTRL_SRV_REG_CFG_INIT;
-#endif
 
 	srv->cfg = cfg;
-#if CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG
 	srv->reg.cfg = reg_cfg;
-#endif
 
 	ctrl_disable(srv);
```

Comment:
This will stop the spec regulator, but is it any thing else that need to reset in the spec regulator? Then running init again the spec regulator will be initialized again, and the reinitialize the timer.